Duplicate identifier 'Subscriber'

Am getting duplicate identifier subscriber issue from past 2 weeks.

![error](https://user-images.githubusercontent.com/52817363/68186536-66854f00-ffca-11e9-98ae-06632f83c2a3.png)

Package.json.

```
{
  "name": "test",
  "version": "0.1.9",
  "author": "Ionic Framework",
  "homepage": "http://ionicframework.com/",
  "private": true,
  "scripts": {
    "clean": "ionic-app-scripts clean",
    "build": "BUILD_ENV=${BUILD_ENV} ionic-app-scripts build",
    "lint": "ionic-app-scripts lint",
    "ionic:build": "ionic-app-scripts build",
    "ionic:serve": "ionic-app-scripts serve",
    "compodoc": "./node_modules/.bin/compodoc -p src/tsconfig.app.json -w -s",
    "test": "karma start ./test-config/karma.conf.js",
    "test-ci": "karma start ./test-config/karma.conf.js --single-run",
    "test-coverage": "karma start ./test-config/karma.conf.js --coverage",
    "e2e": "npm run e2e-update && npm run e2e-test",
    "e2e-test": "protractor ./test-config/protractor.conf.js",
    "e2e-update": "webdriver-manager update --standalone false --gecko false"
  },
  "config": {
    "ionic_generate_source_map": "true"
  },
  "dependencies": {
    "@angular/animations": "5.2.11",
    "@angular/cdk": "^5.2.5",
    "@angular/common": "5.2.11",
    "@angular/compiler": "5.2.11",
    "@angular/compiler-cli": "5.2.11",
    "@angular/core": "5.2.11",
    "@angular/forms": "5.2.11",
    "@angular/http": "5.2.11",
    "@angular/material": "^7.3.5",
    "@angular/platform-browser": "5.2.11",
    "@angular/platform-browser-dynamic": "5.2.11",
    "@ionic-native/app-center-analytics": "^4.18.0",
    "@ionic-native/app-center-crashes": "^4.18.0",
    "@ionic-native/app-version": "^4.17.0",
    "@ionic-native/barcode-scanner": "^4.20.0",
    "@ionic-native/camera": "^4.17.0",
    "@ionic-native/code-push": "^4.20.0",
    "@ionic-native/core": "~4.15.0",
    "@ionic-native/device": "4.7.0",
    "@ionic-native/dialogs": "^4.17.0",
    "@ionic-native/file": "^4.20.0",
    "@ionic-native/image-picker": "^4.17.0",
    "@ionic-native/social-sharing": "^4.17.0",
    "@ionic-native/speech-recognition": "^4.17.0",
    "@ionic-native/splash-screen": "~4.15.0",
    "@ionic-native/status-bar": "~4.15.0",
    "@ionic/pro": "2.0.3",
    "@ionic/storage": "2.2.0",
    "apollo-client": "^2.6.2",
    "aws-amplify": "^1.2.4",
    "aws-amplify-angular": "^3.0.3",
    "aws-appsync": "^1.8.0",
    "cordova-android": "7.1.4",
    "cordova-android-play-services-gradle-release": "^2.0.0",
    "cordova-android-support-gradle-release": "^2.0.1",
    "cordova-browser": "5.0.4",
    "cordova-ios": "4.5.4",
    "cordova-plugin-app-version": "^0.1.9",
    "cordova-plugin-camera": "^4.0.3",
    "cordova-plugin-compat": "^1.2.0",
    "cordova-plugin-device": "^2.0.2",
    "cordova-plugin-dialogs": "^2.0.1",
    "cordova-plugin-file": "^6.0.1",
    "cordova-plugin-file-transfer": "^1.7.1",
    "cordova-plugin-ionic": "^5.3.0",
    "cordova-plugin-ionic-keyboard": "^2.1.3",
    "cordova-plugin-ionic-webview": "^2.2.5",
    "cordova-plugin-speechrecognition": "^1.2.0",
    "cordova-plugin-splashscreen": "^5.0.2",
    "cordova-plugin-statusbar": "^2.4.2",
    "cordova-plugin-telerik-imagepicker": "^2.2.2",
    "cordova-plugin-whitelist": "^1.3.3",
    "cordova-plugin-x-socialsharing": "^5.4.3",
    "cordova-plugin-zip": "^3.1.0",
    "es6-promise-plugin": "^4.2.2",
    "font-awesome": "^4.7.0",
    "graphql-tag": "^2.10.1",
    "ionic-angular": "3.9.2",
    "ionic-long-press": "^1.5.0",
    "ionicons": "3.0.0",
    "ng2-translate": "^5.0.0",
    "phonegap-plugin-barcodescanner": "^8.0.1",
    "primeng": "^5.2.7",
    "rxjs": "5.5.11",
    "sentry-cordova": "^0.15.0",
    "sw-toolbox": "3.6.0",
    "zone.js": "0.8.26"
  },
  "devDependencies": {
    "@compodoc/compodoc": "^1.1.8",
    "@ionic/app-scripts": "3.2.2",
    "@types/jasmine": "^2.8.12",
    "@types/node": "^10.12.10",
    "angular2-template-loader": "^0.6.2",
    "html-loader": "^0.5.5",
    "istanbul-instrumenter-loader": "^3.0.1",
    "jasmine": "^2.99.0",
    "jasmine-spec-reporter": "^4.2.1",
    "karma": "^3.1.1",
    "karma-chrome-launcher": "^2.2.0",
    "karma-coverage-istanbul-reporter": "^2.0.4",
    "karma-jasmine": "^2.0.1",
    "karma-jasmine-html-reporter": "^1.4.0",
    "karma-sourcemap-loader": "^0.3.7",
    "karma-webpack": "^3.0.5",
    "null-loader": "^0.1.1",
    "protractor": "^5.4.1",
    "ts-loader": "^3.5.0",
    "ts-node": "^7.0.1",
    "typescript": "3.3.3333",
    "webpack": "^3.12.0"
  },
  "description": "An Ionic project",
  "cordova": {
    "plugins": {
      "cordova-plugin-statusbar": {},
      "cordova-plugin-device": {},
      "cordova-plugin-splashscreen": {},
      "cordova-plugin-ionic-webview": {
        "ANDROID_SUPPORT_ANNOTATIONS_VERSION": "27.+"
      },
      "cordova-plugin-ionic-keyboard": {},
      "cordova-plugin-speechrecognition": {},
      "cordova-plugin-dialogs": {},
      "cordova-plugin-camera": {},
      "cordova-plugin-file": {},
      "cordova-plugin-file-transfer": {},
      "cordova-plugin-zip": {},
      "cordova-plugin-app-version": {},
      "cordova-plugin-telerik-imagepicker": {
        "PHOTO_LIBRARY_USAGE_DESCRIPTION": "Allow to access your gallery"
      },
      "cordova-plugin-x-socialsharing": {
        "ANDROID_SUPPORT_V4_VERSION": "24.1.1+"
      },
      "cordova-plugin-appcenter-analytics": {},
      "cordova-plugin-appcenter-crashes": {},
      "cordova-android-play-services-gradle-release": {
        "PLAY_SERVICES_VERSION": "15.+"
      },
      "cordova-android-support-gradle-release": {
        "ANDROID_SUPPORT_VERSION": "27.+"
      },
      "cordova-plugin-code-push": {},
      "phonegap-plugin-barcodescanner": {
        "ANDROID_SUPPORT_V4_VERSION": "27.+"
      },
      "cordova-plugin-whitelist": {},
      "sentry-cordova": {}
    },
    "platforms": [
      "ios",
      "android"
    ]
  }
}
```
We gone through https://github.com/aws-amplify/amplify-js/issues/3248 issue and updated the comment but i did't get any reply on that.

after running `npm ls @types/zen-observable` I got 

```
+-- @types/zen-observable@0.8.0
+-- aws-amplify@1.2.4
| +-- @aws-amplify/api@1.2.4
| | `-- @types/zen-observable@0.5.4
| `-- @aws-amplify/pubsub@1.2.4
|   `-- @types/zen-observable@0.5.4
`-- aws-appsync@1.8.1
  +-- apollo-client@2.4.6
  | `-- @types/zen-observable@0.8.0  deduped
  `-- apollo-link-retry@2.2.5
    `-- @types/zen-observable@0.8.0  deduped
```
Uninstalling the root `@types/zen-observable@0.8.0` doesn't fix this
I still receive 
`ERROR in node_modules/@aws-amplify/api/node_modules/@types/zen-observable/index.d.ts:32:14 - error TS2300: Duplicate identifier 'Subscriber'.`

--- a/packages/api/package.json
+++ b/packages/api/package.json
@@ -34,7 +34,7 @@
   },
   "homepage": "https://aws-amplify.github.io/",
   "devDependencies": {
-    "@types/zen-observable": "^0.5.3"
+    "@types/zen-observable": "^0.8.0"
   },
   "dependencies": {
     "@aws-amplify/auth": "^2.1.0",

--- a/packages/api/package.json
+++ b/packages/api/package.json
@@ -33,11 +33,13 @@
     "url": "https://github.com/aws/aws-amplify/issues"
   },
   "homepage": "https://aws-amplify.github.io/",
+  "devDependencies": {
+    "@types/zen-observable": "^0.5.3"
+  },
   "dependencies": {
     "@aws-amplify/auth": "^2.1.0",
     "@aws-amplify/cache": "^2.1.0",
     "@aws-amplify/core": "^2.1.0",
-    "@types/zen-observable": "^0.5.3",
     "axios": "^0.19.0",
     "graphql": "14.0.0",
     "uuid": "^3.2.1",

--- a/packages/pubsub/package.json
+++ b/packages/pubsub/package.json
@@ -34,7 +34,7 @@
   },
   "homepage": "https://aws-amplify.github.io/",
   "devDependencies": {
-    "@types/zen-observable": "^0.5.3",
+    "@types/zen-observable": "^0.8.0",
     "cpx": "^1.5.0"
   },
   "dependencies": {

--- a/packages/pubsub/package.json
+++ b/packages/pubsub/package.json
@@ -34,13 +34,13 @@
   },
   "homepage": "https://aws-amplify.github.io/",
   "devDependencies": {
+    "@types/zen-observable": "^0.5.3",
     "cpx": "^1.5.0"
   },
   "dependencies": {
     "@aws-amplify/auth": "^2.1.0",
     "@aws-amplify/cache": "^2.1.0",
     "@aws-amplify/core": "^2.1.0",
-    "@types/zen-observable": "^0.5.3",
     "graphql": "14.0.0",
     "paho-mqtt": "^1.1.0",
     "uuid": "^3.2.1",

